ci/serverless/README.md

```diff
@@ -25,15 +25,14 @@ The test cases against serverless Elasticsearch cover the following scenarios
 
 #### Logstash
 
-Create Logstash API key for test setup/teardown and elastic_integration filter
+Plugin user
```

Comment:
Nit: 

Can we re-add the `POST /_security/api_key` before the payload? It's helpful for copy pasting into the dev console